libr/util/unum.c

```diff
@@ -10,7 +10,7 @@
 static ut64 r_num_tailff(RNum *num, const char *hex);
 
 R_API void r_num_irand() {
-	srand (r_sys_now ());
+  r_srand (r_sys_now());
```

Comment:
use tabs not spaces

Reply:
add space before (

Reply:
Sorry various projects style mix up :-) Hopefully all addressed newer commit.